lib/sqlalchemy/sql/selectable.py

```diff
@@ -4870,7 +4870,7 @@ def _create(cls, *args, **kw):
           constructs as given, as well as ORM-mapped classes.
 
         """
-        if (args and isinstance(args[0], list)) or kw:
+        if (args and hasattr(args[0], '__iter__')) or kw:
```

Comment:
unfortunately strings have an ` __iter__` method also, so this interferes with an error condition we have for the newer style:

```
>>> from sqlalchemy import select
>>> select("foo")
Traceback (most recent call last):
...
sqlalchemy.exc.ArgumentError: Textual column expression 'f' should be explicitly declared with text('f'), or use column('f') for more specificity

```

one way to sneak around this, which we would need to really accept "iterable", is to check for a string-based method also:

    if hasattr(x, "__iter__") and not hasattr(x, "strip")

we have some other code that uses that one specifically.